civihr-install

```diff
@@ -885,6 +885,9 @@ function civihr_install_site(){
 
     drush vset --format=integer node_export_reset_path_webform 0
 
+    ## Disable error logging in Drupal
+    drush vset -y error_level 0
```

Comment:
After a bit of discussion Davi explained that this was to hide some warnings that are shown during the setup process. Ideally at some stage we can fix these instead of hiding them, but for now it seems this is an intentional choice to hide them so approving the PR